.github/workflows/live-watch-fe.yml

```
@@ -0,0 +1,102 @@
+      - 'packages/core/**'
+      - 'packages/ui/**'
+      - '.github/workflows/live-watch-fe.yml'
+
+jobs:
+  dev_deploy:
+    runs-on: ubuntu-latest
+    environment:
+      name: dev
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)